libshpool/src/lib.rs

```diff
@@ -169,6 +169,14 @@ impl Args {
 /// Run the shpool tool with the given arguments. If hooks is provided,
 /// inject the callbacks into the daemon.
 pub fn run(args: Args, hooks: Option<Box<dyn hooks::Hooks + Send + Sync>>) -> anyhow::Result<()> {
+    if matches!(
+        (&args.command, env::var(consts::PROMPT_SENTINEL_FLAG_VAR).as_deref()),
+        (Commands::Daemon, Ok("yes"))
+    ) {
+        println!("{}", consts::PROMPT_SENTINEL);
```

Comment:
I don't think that will be a problem because when run in this mode the daemon immediately exits, and it does nothing but print the sentinel, so nothing else can be interleaved in the output.

Reply:
sounds good!